include/deal.II/base/discrete_time.h

```diff
@@ -21,13 +21,20 @@
 DEAL_II_NAMESPACE_OPEN
 
 /**
- * Provides a means to keep track of the simulation time in a time-dependent
- * simulation. It manages stepping forward from an initial time to a final
- * time. It also allows adjusting the time step size during the simulation.
- * It is guaranteed that at all times the current simulation time is in the
- * closed interval between the start time and the end time.
+ * This class provides a means to keep track of the simulation time in a
+ * time-dependent simulation. It manages stepping forward from an initial time
+ * to a final time. It also allows adjusting the time step size during the
```

Comment:
```suggestion
 * time-dependent simulation. It manages stepping forward from a start time $T_{\text{start}}$
 * to an end time $T_{\text{end}}$. It also allows adjusting the time step size during the
```
We can use the start-time/end-time idiom to be consistent with our function names. What do you think?